modules/sops/default.nix

```
@@ -242,6 +243,17 @@ in {
       '';
     };
 
+    useTmpfs = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Use tmpfs in place of ramfs for secrets storage.
+
+        *WARNING*
+        Enabling this option has the potential to write secrets to disk unencrypted if the tmpfs volume is written to swap. Do not use unless absolutely necessary.
```

```suggestion
        Enabling this option has the potential to write secrets to disk unencrypted if the tmpfs volume is written to swap. Do not use unless absolutely necessary.
        
        Consider enabling swap encryption when using this option:
        
        ```
        swapDevices = [{
          device = "/dev/sdXY";
          randomEncryption.enable = true; 
        }];
        ```
```